### ISSUE TITLE
JODI Oil Database (?)

### ISSUE THREAD

**Issue by rufuspollock** [datasets/awesome-data#4]
http://www.jodidata.org/database/access-database.aspx

Not quite sure what is in there but seems to be oil reserves etc

**Comment by sanh2o** [comment]
They have 13 product, 14 flows data in 3 different units per their site description starting from 2002 and updated monthly. These can be downloaded as txt / csv or .ivt (?) and there exist 4 or 5 different files with one containing indexes and explanations.

**Comment by rgrp** [comment]
@sanh2o could you summarize here what exactly the 13 product + 14 flows (+ 3 different units are) here (or in a gist or spreadsheet if more useful). We can then make a decision about what to pull.

**Comment by sanh2o** [comment]
Summary of Jodi data categories 
<script src="https://gist.github.com/sanh2o/4a45e93ac52c8c9266dd.js"></script>

**Comment by pdehaye** [comment]
Giving this a bump. Any decision taken?

**Comment by rgrp** [comment]
@sanh2o do you have any stats on each item or a description.
